deploy/roles/gocd/tasks/main.yaml

```diff
@@ -58,36 +57,41 @@
     mode: 0755
   notify:
     - restart go-agent
+  tags: gocd
 
 - name: Replace go-agent file
   template:
     src: go-agent
     dest: /etc/default/go-agent
   notify:
     - restart go-agent
+  tags: gocd
 
 - name: Start go-agent
   service:
     name: go-agent
     state: started
+  tags: gocd
 
 - name: Set recursive admin permissions on log folders
   acl:
     path: "{{ item }}"
     entity: "{{ restricted_group_name }}"
     etype: group
-    permissions: rX
+    permissions: rx
```

Comment:
@stirlingv This shoudl be `rX` I'm pretty sure, the capital X makes sure that only folders are set as executable, not files, which is what we want.

Reply:
👍